solr/core/src/java/org/apache/solr/search/JoinQParserPlugin.java

```diff
@@ -160,23 +176,40 @@ JoinParams parseJoin(QParser qparser) throws SyntaxError {
     }
   }
 
+  @Override
+  public void init(NamedList args) {
+    routerField = (String) args.get("routerField");
+    solrUrlWhitelist = new HashSet<>();
+    if (args.get("solrUrlWhitelist") != null) {
+      //noinspection unchecked
+      for (String s : (List<String>) args.get("solrUrlWhitelist")) {
+        if (!StringUtils.isEmpty(s))
```

Comment:
This is fine, really, but personally I'd just go with 1 line of code -- `solrUrlWhiteList.addAll((List<String>) args.get("solrUrlWhitelist"))` because the impact of an empty string here is safe.

Reply:
Sounds good to me - I made this change while updating the property name.